allennlp/training/metrics/span_based_f1_measure.py

```diff
@@ -111,91 +112,23 @@ def __call__(self,
                 # It is possible to call this metric with sequences which are
                 # completely padded. These contribute nothing, so we skip these rows.
                 continue
-            prediction_spans = self._extract_spans(sequence_prediction[:length].tolist())
-            gold_spans = self._extract_spans(sequence_gold_label[:length].tolist())
+
+            predicted_string_labels = [self._label_vocabulary[label_id]
+                                       for label_id in sequence_prediction[:length].tolist()]
+            gold_string_labels = [self._label_vocabulary[label_id]
+                                  for label_id in sequence_gold_label[:length].tolist()]
+            prediction_spans = bio_tags_to_spans(predicted_string_labels, self._ignore_classes)
```

Comment:
Above you have `predicted` and here you have `prediction` - can you be consistent between those two?  No strong preference on which one you should use.